### ISSUE TITLE
TypeError: Cannot read property 'click' of undefined

### ISSUE THREAD

**Issue by jeffposnick** [GoogleChrome/ioweb2016#793]
There's a decent number of the following with Chrome 49/50 logged in GA error reporting:

```
TypeError: Cannot read property 'click' of undefined
at paper-tabs.Polymer.IronMenubarBehaviorImpl._onDownKey (https://events.google.com/io2016/elements/critical.html?2016-04-21:10416:23)
at paper-tabs.Polymer.IronA11yKeysBehavior._triggerKeyHandler (https://events.google.com/io2016/elements/critical.html?2016-04-21:10057:27)
at paper-tabs.Polymer.IronA11yKeysBehavior._onKeyBindingEvent (https://events.google.com/io2016/elements/critical.html?2016-04-21:10041:18)
```

and

```
TypeError: Cannot read property 'click' of undefined
at paper-tabs.Polymer.IronMenubarBehaviorImpl._onUpKey (https://events.google.com/io2016/elements/critical.html?2016-04-21:10411:23)
at paper-tabs.Polymer.IronA11yKeysBehavior._triggerKeyHandler (https://events.google.com/io2016/elements/critical.html?2016-04-21:10057:27)
at paper-tabs.Polymer.IronA11yKeysBehavior._onKeyBindingEvent (https://events.google.com/io2016/elements/critical.html?2016-04-21:10041:18)
```

Assigning to @robdodson since I remember him touching some of that code for a11y tinkering. No good deed, etc.
